Rakefile

```diff
@@ -1,11 +1,39 @@
-desc 'Clean up generated site'
+#!/usr/bin/env rake
+
+desc "Clean up generated site"
 task :clean do
-  sh 'rm -rf _site'
+  sh "rm -rf _site"
 end
 
+desc "Runs the tests!"
 task :test do
-  sh 'bundle exec jekyll build'
+  require "html-proofer"
 
+  File.readlines("script/writewell-files.txt").each do |line|
+    HTMLProofer.check_file(line.delete!("\n"), {
+      :empty_alt_ignore => true,
+      :external_only => true,
+      :url_ignore  => [
+        %r{https?://github.com/new},
+        /\.github\.io/,
+        /\.gitter\.im/,
+        /atom\.io/,
+        /git-scm\.com/,
+        /github\.com/,
+        /mademistakes\.com/,
+        /nasa\.gov/,
+        /travis-ci\.org/,
```

Comment:
@brianamarie, if you think it's necessary, we can add `/vimeo\.com/` here as an exception for the link checker.